mp/src/game/shared/momentum/mom_rocket.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#ifndef CLIENT_DLL
+#include "smoke_trail.h"
```

Comment:
Move this down to line 10, take advantage of the `#else` of the already done `#ifdef CLIENT_DLL`